controller/app.controller.js

```diff
@@ -0,0 +1,9 @@
+const { selectTopic } = require("../models/app.model");
```

Comment:
Rename this to selectTopics, please! It returns multiple topics, and our naming should reflect that.